app/migration/mappers/programme_type_mapper.rb

```diff
@@ -0,0 +1,19 @@
+module Mappers
+  class ProgrammeTypeMapper
```

Comment:
Could this be named `TrainingProgrammeTypeMapper` or `InductionProgrammeTypeMapper` just to be super clear?